x/uibc/quota/grpc_query.go

```diff
@@ -51,11 +51,22 @@ func (q Querier) AllOutflows(goCtx context.Context, _ *uibc.QueryAllOutflows) (
 	*uibc.QueryAllOutflowsResponse, error,
 ) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	o, err := q.Keeper(&ctx).GetAllOutflows()
+	k := q.Keeper(&ctx)
+	o, err := k.GetAllOutflows()
 	if err != nil {
 		return nil, err
 	}
-	return &uibc.QueryAllOutflowsResponse{Outflows: o}, nil
+	tokenSymbols := map[string]string{}
+	for _, t := range k.leverage.GetAllRegisteredTokens(ctx) {
+		tokenSymbols[t.BaseDenom] = t.SymbolDenom
+	}
+	outflows := make([]uibc.DecCoinSymbol, len(o))
+	for i := range o {
+		outflows[i].Denom = o[i].Denom
+		outflows[i].Amount = o[i].Amount
+		outflows[i].Symbol = tokenSymbols[o[i].Denom]
+	}
```

Comment:
We have duplicate Symbol Denoms , so how front-end will know this is the `tether usdt` and normal `usdt` , May be we should  include base_denom to outflows  because `base_denom` are unique for each token registry record

Reply:
I think, this is solved by the `Denom`.

Reply:
Got it 👍